<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseView</class>
 <widget class="QWidget" name="DatabaseView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1353</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>起始时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit_startTime">
       <property name="date">
        <date>
         <year>2024</year>
         <month>8</month>
         <day>11</day>
        </date>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>结束时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit_endTime">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2024</year>
         <month>8</month>
         <day>12</day>
        </datetime>
       </property>
       <property name="date">
        <date>
         <year>2024</year>
         <month>8</month>
         <day>12</day>
        </date>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_TimeSelect">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>时间筛选标志</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_addData">
       <property name="text">
        <string>添加数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_outExcel">
       <property name="text">
        <string>导出表格</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_freshenDB">
       <property name="text">
        <string>刷新数据库</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OpenNewDB">
       <property name="text">
        <string>打开附加数据库</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_addDB">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>附加库标志</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>表格选择:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cB_tableKind">
       <item>
        <property name="text">
         <string>指令发送记录</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1553B慢速遥测记录组1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1553B慢速遥测记录组2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1553B慢速遥测记录组3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CAN遥测记录</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView_DB"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
